src/windowsdesktop/src/bundle/bundle.wxs

```diff
@@ -0,0 +1,241 @@
+              Result="exists"
+              Path="[ProgramFiles64Folder]dotnet\dotnet.exe"/>
+    <util:DirectorySearch Id="DotnetExeLocation_arm64"
+              After="DotnetExeSearch_arm64"
+              Condition="DotnetExeExists_arm64"
+              Variable="DOTNETHOME_ARM64"
+              Path="[ProgramFiles64Folder]dotnet"/>
+    <?endif?>
+
+    <!--
+        When installing the SDK bundle to a custom location using the commandline parameters, it is intended, not mandatory, that
+        both "DOTNETHOME_X86" and "DOTNETHOME_X64" should be used on the commandline and should take this convention:
+            DOTNETHOME_X86=<InstallFolder>\x86
+            DOTNETHOME_X64=<InstallFolder>\x64
+        Example:
+            dotnet-sdk-3.0.100-alpha1-009719-win-x64.exe /install DOTNETHOME_X64="D:\dotnet\x64" DOTNETHOME_X86="D:\dotnet\x86" /log "installation.log" /quiet /norestart
+    -->
+    <Variable Name="DOTNETHOME_X86" bal:Overridable="yes" />
+    <Variable Name="DOTNETHOME_X64" bal:Overridable="yes" />
+    <Variable Name="DOTNETHOME_ARM64" bal:Overridable="yes" />
+    <Variable Name="DOTNETHOME" Type="formatted" Value="[DOTNETHOME_$(PlatformToken)]" bal:Overridable="no" />
+    <Variable Name="BUNDLEMONIKER" Type="string" Value="$(SdkBrandName)" bal:Overridable="no" />
+    <Variable Name="DOTNETSDKVERSION" Type="string" Value="$(Version)" bal:Overridable="no" />
+    <Variable Name="DOTNETRUNTIMEVERSION" Type="string" Value="$(DotNetRuntimeVersion)" bal:Overridable="no" />
+    <Variable Name="ASPNETCOREVERSION" Type="string" Value="$(AspNetCoreVersion)" bal:Overridable="no" />
+    <Variable Name="WINFORMSANDWPFVERSION" Type="string" Value="$(WinFormsAndWpfVersion)" bal:Overridable="no" />
+    <Variable Name="DOTNETHOMESIMILARITYCHECKOVERRIDE" Type="string" Value="" bal:Overridable="yes" />
+    <Variable Name="VERSIONMAJOR" Type="string" Value="$(MajorVersion)" bal:Overridable="no" />
+    <Variable Name="VERSIONMINOR" Type="string" Value="$(MinorVersion)" bal:Overridable="no" />
+    <Variable Name="MINIMUMVSVERSION" Type="string" Value="$(MinimumVSVersion)" bal:Overridable="no" />
+
+    <Chain DisableSystemRestore="yes" ParallelCache="yes">
+      <!-- Rollback boundary ensures transaction safety for all MSI packages -->
+      <RollbackBoundary Id="WindowsDesktopBoundary" />
+      
+      <!-- .NET Core components must be installed in dependency order -->
+      <!-- These are conditionally included only if the MSI files exist -->
+      
+      <!-- Host is the base component that everything else depends on -->
+      <?if $(var.IncludeRuntimeMSIs) = true ?>
+      <MsiPackage 
+        Id="DotNetHost"
+        SourceFile="$(var.DotNetRedistHostInstaller)"
+        Vital="yes"
+        Cache="keep"
+        DisplayName="Microsoft .NET Host"
+        InstallCondition="NOT DotNetHostInstalledVersion$(var.PlatformToken) OR DotNetHostInstalledVersion$(var.PlatformToken) &lt; v$(var.DotNetRuntimeVersion)">
```

Comment:
What's the reason for these conditions. Bundles automatically detect and install the MSIs if not present and if they are, they'll bump the reference count accordingly. Having a condition will result in skipping an MSI package, so if you installed the runtime + desktop - desktop bundle, the runtime MSIs would get removed because there is no reference count.